### DISCUSSION TITLE
Should we drop proviola support?

### DISCUSSION THREAD

**Discussion by Alizter** [HoTT/HoTT#1726] [General]
I would suggest we drop support for proviola:

Reasons for:
- Adds complexity to makefile
- Takes a long time in CI
- Is no longer maintained upstream
- Randomly breaks due to syntax changes or random occurrences in GitHub runners
- Entire functionality subsumed by alectryon (IIANM)

Reasons against:
- We are one of the only projects to user it, by dropping support we inadvertently kill it

Please suggest any other reasonings I have missed.

cc @JasonGross

**Comment by SkySkimmer** [comment]
>Is no longer maintained upstream

Sounds like it's already dead, so we can't kill it

**Comment by jdchristensen** [comment]
I think we should drop it.  I compared alectryon and proviola, and they are fairly similar.  One handy thing that proviola has is that if you click on an identifier, it jumps to the definition.  Can alectryon do that?
